repo/fsrepo/migrations/mfsr.go

```diff
@@ -49,13 +49,36 @@ func (rp RepoPath) CheckVersion(version string) error {
 	return nil
 }
 
-func (rp RepoPath) WriteVersion(version string) error {
+func (rp RepoPath) WriteVersion(version int) error {
 	fn := rp.VersionFile()
-	return ioutil.WriteFile(fn, []byte(version+"\n"), 0644)
+	return ioutil.WriteFile(fn, []byte(fmt.Sprintf("%d\n", version)), 0644)
 }
 
 type VersionFileNotFound string
 
 func (v VersionFileNotFound) Error() string {
 	return "no version file in repo at " + string(v)
 }
+
+func TryMigrating(tovers int) error {
+	if !YesNoPrompt("run migrations automatically? [y/n]") {
+		return fmt.Errorf("please run the migrations manually")
+	}
+
+	return RunMigration(tovers)
+}
```

Comment:
Sorry for creeping up on you with this feedback one day before the planned release :( :)

This should be part of the daemon command. With previous migrations, the daemon would abort with a non-zero exit code, now it'll just hang waiting for input. It'll be useful to have a `-migrate=true|false` option which assumes yes/no and omits the prompt. (Actually it's really important for server environments.)